src/translations/pl.js

```diff
@@ -110,7 +108,17 @@ export const pl = {
       connection: 'Stwórz nowe połączenie',
       back: 'Ukryj kafelek',
     },
-    unrecognizedProblem: 'Coś poszło nie tak...',
+    error: {
+      unrecognizedProblem: 'Coś poszło nie tak. Skontaktuj się z administratorem.',
+      unknownStatusError: 'Nieznany status błędu. Skontaktuj się z administratorem.',
+      unableToShowErrorMessage: 'Problem z wyświetleniem informacji o błędzie. Skontaktuj się z administratorem.',
+      validationError: 'Problem z walidacją. Skontaktuj się z administratorem.',
+      errorIsNull: 'Brak odpowiedzi z serwera. Skontaktuj się z administratorem.',
+      authorization: 'Problem z autoryzacją. Wyloguj się i zaloguj ponownie.',
```

Comment:
Im a bit confused. We speak about authorization when user usually is authenticated but his privileges are not enough to perform specific task/request.
We speak about authentication when we want to know what user we deal with.
Shouldn't be these two statement swapped?

Reply:
Yes